packages/instant/src/util/buy_quote_updater.ts

```diff
@@ -49,8 +50,9 @@ export const buyQuoteUpdater = {
                 } else {
                     throw error;
                 }
+            } else {
+                errorReporter.report(error);
```

Comment:
hmm do we want to report here if we are intentionally suppressing errors? These are errors that we would be catching and swallowing otherwise

Reply:
Good callout.  I can update the code, but wanted to make sure we are on the same page wrt to the desired implementation here:

When this `catch` is hit, there are two types of errors: "Known / Expected" errors (i.e. not enough ZRX available), which we want to swallow, not report to rollbar, and flash to the user if we are not in a heartbeat.  Then there are "Unexpected" errors, which we want to always report to rollbar, regardless of if we are in a heartbeat or not.